thirdeye/thirdeye-frontend/app/pods/custom/composite-animalies-table/resolution/component.js

```diff
@@ -0,0 +1,18 @@
+import Component from "@ember/component";
+import { set } from '@ember/object';
+
+export default Component.extend({
+    didReceiveAttrs() {
```

Comment:
This override is not needed as you are not doing any additional changes in this function.

Reply:
hm... for some reason, `data` was coming *undefined* 
I'll take another look